src/_help/getting-started/concepts.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Main concepts
+---
+
+- TOC
+{:toc}
+
+Before we proceed further, we wanted to share some vocabulary to clarify what we are talking about throughout this documentation.<br>
+The terminology used here aims to be as accurate as possible, but you may encounter different terms elsewhere. We have chosen to use the most common and meaningful terms.
+
+## API definition
+
+An API definition is the representation of an API written in compliance with an existing specification format.<br>
+It is sometimes referred to as an API schema, API contract, or even API specification.
+
+As of today, Bump.sh supports the following specifications: Swagger, [OpenAPI](https://spec.openapis.org/oas/v3.1.0), and [AsyncAPI](https://www.asyncapi.com/docs).<br>
+Each standard has its prerequisites, and while your API definition doesn't need to be 100% compliant with them to be used on Bump.sh, it should generally follow their structure.
+
+We provide a detailed introduction to API definitions [through this guide](https://docs.bump.sh/guides/api-basics/api-contracts-extended-introduction/).
+
+## API contract
+
+An API contract represents the use of an API definition when it defines an agreement, a contract between API developers, its consumers, how it is used, and the tools that surround it.
+
+## API document
+
+An API document refers to the file containing an API definition.<br>
+Bump.sh supports YAML and JSON formats for API documents.
```

Comment:
```suggestion
An API document refers to the file containing an API definition.
```

As we'll soon add WSDL (XML) and GraphQL schemas, we don't want to start listing the formats here, imo.